en_US.json

```diff
@@ -221,7 +223,7 @@
 	"certificate.confirm.cancel.edit.certificate": "Do you want to discard the certificate update?",
 	"certificate.create.success.modal.content": "The certificate has been successfully saved.",
 	"certificate.create.success.modal.title": "Certificate saved!",
-	"certificate.create.title": "Create a Certificate",
+	"certificate.create.title": "Create a Server Certificate",
```

Comment:
We don't have to change this. We can take "Create a Certificate" as creating server certificates, while "Create a CA Certificate" as creating CA certificates.
Although we distinguish server certificates and CA certificates, what our customers will manage would still be server certificates mostly.